gamestonk_terminal/stocks/fundamental_analysis/dcf_model.py

```diff
@@ -224,234 +224,187 @@ def others_in_sector(ticker: str, sector: str, industry: str) -> List[str]:
     industry = industry.replace("—", " - ")
     industry = industry.replace("/", " ")
 
-    sister_ticks = list(
-        fd.select_equities(country="United States", sector=sector, industry=industry)
+    sister_ticks = fd.select_equities(
+        country="United States", sector=sector, industry=industry
     )
+
+    # This filters sisters to match marekt cap and removes ticker analyzed
     if ticker in sister_ticks:
-        sister_ticks.remove(ticker)
+        market_cap = sister_ticks[ticker]["market_cap"]
+        sister_ticks.pop(ticker, None)
+        sister_ticks = {
+            k: v for (k, v) in sister_ticks.items() if v["market_cap"] == market_cap
+        }
+    sister_ticks = list(sister_ticks)
```

Comment:
It might be beneficial to include a parameter that allows you to also select 'all' market caps. Sometimes the industry can be relatively small (or perhaps market cap is less important) that you would want to include all companies.